routes/index.js

```diff
@@ -4,10 +4,12 @@ const router = express.Router();
 const {authRouter} = require("./authRouter");
 const {productRouter} = require("./productRouter")
 const reviewRouter = require("./reviewRouter");
+const cartRouter = require("./cartRouter")
 
 router.use("/product",productRouter);
 router.use("/auth", authRouter);
 router.use("/review", reviewRouter.router);
+router.use("/cart", cartRouter.router)
```

Comment:
```suggestion
router.use("/carts", cartRouter.router)
```
endpoint 주소가 `:3000/carts`만 되어도 충분할 것 같아요.